src/config/db.js

```diff
@@ -1,8 +1,17 @@
 const mongoose = require('mongoose');
 require('../utils/env');
 
-const uri = process.env.MONGODB_URI;
+let uri = process.env.MONGODB_URI;
 
+/**
+ * Uncomment this line of code to use a local MongoDB server
+ */
+
+  if (process.env.NODE_ENV !== 'production') {
```

Comment:
the production version will use the URI in the `.env` file.

Reply:
I know I never made any significant change. Just moved console.log(uri) from top to bottom so it print the rightful uri instead of the misleading one

Reply:
Ok. It seems your tree is a bit behind

Reply:
You dont mean it lol. Something I pulled and fixed conflicts before sending PR. Hold on a moment lemme get you screenshot

Reply:
![image](https://user-images.githubusercontent.com/32165413/85232161-02968600-b3f5-11ea-8bd7-830a1e1567a6.png)

The 3 commit behind must be from your newly merged